keyboards/treadstone32/keymaps/default/rules.mk

```diff
@@ -0,0 +1,26 @@
+
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
+MOUSEKEY_ENABLE =  yes       # Mouse keys(+4700)
```

Comment:
All of the rules here should be set at the keyboard level instead. Putting these settings here and not there limits and changes the functionality for users who may use QMK Configurator to program their keyboards.